### PR TITLE
Updates config

### PR DESCRIPTION
**Pull request by Amelia0010**

### CODE REVIEW

File: config/motd.txt [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">We have a support ticket system at Yogstation. If you have an issue, please file a ticket by pressing <b>F1</b> or going to the <b>admin tab</b> and clicking <b>Ahelp</b>. An admin will help you as soon as possible.</span>
 
-<h1>Welcome to Space Station 13!</h1>
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">We are constantly seeking dedicated new <span style="font-weight: bold; color: #dd4400;">admins</span>, <span style="font-weight: bold; color: #dd4400;">coders</span>, <span style="font-weight: bold; color: #dd4400;">spriters</span> and <span style="font-weight: bold; color: #dd4400;">mappers</span> to improve our server. Without them, you wouldn't have a server to play on. If you feel you may be a good addition to our team, please apply:</span>
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">
+ ¤ <a href="http://forums.yogstation.net/index.php?forms/admin-application-form.4/respond">Admin Application</a>
+ ¤ <a href="http://forums.yogstation.net/index.php?forms/coder-application-form.12/respond">Coder / Spriter / Mapper Application</a>
```
Comment (ThatLing) [inline config/motd.txt:7]: This is outdated and should probably be rewritten

File: config/whitelist.txt [hunk 2]
```diff
@@ -2,8 +2,4 @@
 # Basically, a list of ckeys who make it past user connection                                 #
 # Case is not important for ckey.                                                             #
 ###############################################################################################
-#Optimumtact
-#kevinz000
-#Iamgoofball
-#Rshoe95
-#That one penguin who became an admin
+#Nichlas0010
```
Comment (ThatLing) [inline config/whitelist.txt:5]: Nice meme

File: config/game_options.txt [hunk 3]
```diff
@@ -37,8 +37,8 @@ EMOJIS
 ## To speed things up make the number negative, to slow things down, make the number positive.
 
 ## These modify the run/walk speed of all mobs before the mob-specific modifiers are applied.
-RUN_DELAY 1
-WALK_DELAY 4
+RUN_DELAY 2
+WALK_DELAY 5
```
Comment (ThatLing) [inline config/game_options.txt]: These should be left at their default values

**Review by ThatLing — DISMISSED** [review]

File: config/game_options.txt (hunk 3 above)
Comment (AsV9) [reply config/game_options.txt]: :thinking:

Comment (Amelia0010) [reply config/game_options.txt]: but tg is sooo faaaast

File: config/whitelist.txt (hunk 2 above)
Comment (Amelia0010) [reply config/whitelist.txt:5]: whitelist isn't used anyways

File: config/motd.txt (hunk 1 above)
Comment (Amelia0010) [reply config/motd.txt:7]: meh, I just ported old one over, someone can rewrite it if they feel like it

Comment (ThatLing) [reply config/motd.txt:7]: Fair enough

**Comment by AsV9** [comment]
looks good to me

**Pull request merged**